tests/smoke-tests/test-historic-compatibility.ts

```diff
@@ -0,0 +1,83 @@
+import "@moonbeam-network/api-augment";
+import { expect } from "chai";
+import { describeSmokeSuite } from "../util/setup-smoke-tests";
+import { NetworkTestArtifact, tracingTxns } from "../util/tracing-txns";
+import Bottleneck from "bottleneck";
+import { providers } from "ethers";
+
+const debug = require("debug")("smoke:historic-compatibility");
+const limiter = new Bottleneck({ maxConcurrent: 10, minTime: 100 });
+const httpEndpoint = process.env.HTTP_URL;
+
+describeSmokeSuite(`Verifying historic compatibility...`, async (context) => {
+  let traceStatic: NetworkTestArtifact;
+  before("Loading tracing static data", async function () {
+    const chainId = (await context.polkadotApi.query.ethereumChainId.chainId()).toString();
+    debug(`Running tracing tests against chainId ${chainId}.`);
+    traceStatic = tracingTxns.find((a) => a.chainId.toString() == chainId);
+
+    if (!traceStatic) {
+      debug(`No test data available for ChainId ${chainId}, skipping test.`);
+      this.skip();
+    }
+  });
+
+  it("can debugTrace for all previous runtimes", async function () {
+    if (httpEndpoint == null) {
```

Comment:
What happens if I execute this script with
```
HTTP_URL= WSS_URL="foobar" ts-node test-historic-compatibility
```

Is this a null or empty string?

Reply:
Good point